src/Icons/tests/Integration/Twig/UXIconExtensionTest.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Icons\Tests\Integration\Twig;
+
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+use Symfony\Component\DomCrawler\Crawler;
+use Twig\Environment;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+final class UXIconExtensionTest extends KernelTestCase
+{
+    public function testRenderIcons(): void
+    {
+        $output = self::getContainer()->get(Environment::class)->createTemplate(<<<TWIG
+            <ul class="svg">
+                <li id="first">{{ ux_icon('user', {class: 'h-6 w-6'}) }}</li>
+                <li id="second">{{ ux_icon('user') }}</li>
+                <li id="third">{{ ux_icon('sub:check') }}</li>
+                <li id="forth">{{ ux_icon('sub/check') }}</li>
```

Comment:
I say let's pick one syntax and forbid other.

(or we'll have to deal with conversion/normalizations in many places) 

wdyt ?

Reply:
I want to choose `:` as the standard separator but in this case, I can't really forbid `/` unless I want to explicitly throw an exception if the name contains it. I don't really want to do that. I think this would be a documentation thing.

The cache key remains the same whether `:` or `/` is used.

Reply:
I see no problem to have a regexp basic check (that will be necessary to avoid some caracters, weird pathes, etc..)

Reply:
If name is handled as an "identifier" i do believe there should be only one syntax, but open to discussion.

Reply:
Okay, I guess do we even want `:` has a directory separator? I chose this because that's what we do for twig components so it's familiar.

Reply:
For ID / classes we'll need a way to distinct all icons from a packages, and i like the "--" method, as it's Stimulus one too.

I send you a PR in that sense.

For me the "path separator" is an implementation details, what we need is something that says "this icon in this package/set/..;"